src/sqlparser/src/ast/mod.rs

```diff
@@ -1514,6 +1514,7 @@ pub enum Statement {
     CreateDatabase {
         db_name: ObjectName,
         if_not_exists: bool,
+        user_specified: Option<ObjectName>,
```

Comment:
Why not just naming it `owner`? 😄

Reply:
👍  fixed together with create schema statement.